bin/cli.js

```diff
@@ -10,12 +10,6 @@
 
 /* eslint-env node */
 
-require('./cli-runner')
-  .run(process.argv.slice(2).join(' '))
-  .catch(e => {
-    // eslint-disable-next-line no-console
-    console.error(e.message);
-    if (e.yargs) e.yargs.showHelp();
-    // Rethrow so that there's a non-zero exit code
-    throw e;
-  });
+require('./cli-runner').run(
+  process.argv.map(x => x.split(' ')).reduce((x, y) => x.concat(y))
```

Comment:
Why is this needed?

Reply:
It is because of the test files. They pass the arguments like 'dir --test` instead of tokenizing them which breaks sade.

Reply:
I think we should pass Array<string> from the tests to match parameter types under regular usage. As it stands, I'm not sure it does the right thing if we pass something like `--dir="/Program Files/something"`

Reply:
It does the right thing. But yes, I was wondering if I should refactor the tests or refactor the cli code for this check. I am pushing another commit with the changes.